intelmq/bots/outputs/files/output.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
```

Comment:
The thought behind naming was that "file" bot writes all event into one file, whereas "files" bot writes onto separate, distinct files. Anything different sounded long or weird, however no strong opinions, if there are any ideas. :)

Concerning merging - I'm a bit afraid merged bot would get hairy and even more confusing - now they are two, but with reasonably simple and understandable/reviewable code, and straightforward set of (some same, some distinct) configuration options.